style.css

```diff
@@ -64,25 +61,200 @@
   background: #1a2236;
 }
 
-button {
+.about-container button {
   width: 288px;
   height: 50px;
-  left: 16px;
-  top: 390px;
-  background: #1a2236;
+  background-color: #1a2236;
   font-family: Poppins, Arial, Helvetica, sans-serif;
   font-style: normal;
   font-weight: 500;
   font-size: 16px;
   line-height: 24px;
   text-align: center;
   color: rgb(255, 255, 255);
+  border-style: none;
 }
 
 .header-img-bottom {
-  position: absolute;
   width: 320px;
   height: 160px;
-  left: 0;
-  top: 472px;
+  margin-left: 0;
+}
+
+/* projects */
+.union {
+  width: 20.57px;
+  height: 15.35px;
+  margin-left: 166px;
+  background: #fff;
+}
+
+.project-img {
+  width: 320px;
+  height: 186px;
+  margin-left: 0;
+  border: 0.333333px solid rgba(255, 255, 255, 0.1);
+}
+
+.project-lg {
+  width: 320px;
+  height: 156px;
+  margin-top: -8px;
+  background: #1a2236;
+  border: 0.333333px solid rgba(255, 255, 255, 0.1);
+  backdrop-filter: blur(1px);
+}
+
+.project-lg-1 {
+  width: 264px;
+  height: 16px;
+  margin-left: 20px;
+}
+
+.project-lg-1 h3 {
+  font-family: Poppins, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 20px;
+  line-height: 80%;
+  color: #fff;
+}
+
+.project-btn-1 {
+  background: #98dede;
+  border-style: none;
+}
+
+.btn-p {
+  width: 49px;
+  height: 8px;
+  font-family: Poppins, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 10.3334px;
+  line-height: 80%;
+  color: #1a2236;
+}
+
+#project-view {
+  margin-top: 20px;
+  background-color: #1a2236;
+  border: 1px solid #fff;
+}
+
+.project-view-union {
+  width: 124px;
+  height: 24px;
+  font-family: Poppins, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  text-align: center;
+  color: #fff;
+}
+
+/* about */
+.about {
+  margin-left: 16px;
+  margin-right: 16px;
+}
+
+.about-me {
+  width: 288px;
+  height: 278px;
+}
+
+.about-me h1 {
+  width: 134px;
+  height: 50px;
+  font-family: Poppins, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 50px;
+  line-height: 100%;
+  color: #1a2236;
+}
+
+.about-me p {
+  width: 288px;
+  height: 126px;
+  font-family: Poppins, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 21px;
+  color: #1a2236;
+}
+
+#about-view {
+  width: 288px;
+  height: 50px;
+  background: #1a2236;
+}
+
+.project-view-btn {
```

Comment:
- [ ] Let's make sure to implement and follow the button styling: Active, Hover and Pressed
![image](https://user-images.githubusercontent.com/46839548/153730241-854c44ac-5d79-428c-b040-89ba81d10391.png)